docs/openj9_support.md

```diff
@@ -72,26 +71,93 @@ show the minimum build configurations that we have tested to build an OpenJDK bi
+
+
+| Linux                                 |   x32  |  x64   | ppc64le    | Z31  | Z64 |
+|---------------------------------------|--------|--------|------------|------|------|
+| Centos 6                              |   Y    |   Y    |     Y      |   N  |  N   |
+| Centos 7                              |   Y    |   Y    |     Y      |   N  |  N   |
+| Red Hat Enterprise Linux (RHEL) 6     |   Y    |   Y    |     Y      |   Y  |  Y   |
+| RHEL 7                                |   Y    |   Y    |     Y      |   Y  |  Y   |
+| SUSE Linux Enterprise Server (SLES) 12|   Y    |   Y    |     Y      |   Y  |  Y   |
+| Ubuntu 16.04                          |   Y    |   Y    |     Y      |   N  |  Y   |
+| Ubuntu 18.04                          |   Y    |   Y    |     Y      |   N  |  Y   |
+
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i><span class="sr-only">Note</span> **Note:** Not all of these distributions are tested, but Linux distributions that have a
+minimum glibc version 2.12 are expected to function without problems.
+
+| Windows                    |  x32   |  x64  |
+|----------------------------|--------|-------|
+| Windows 10                 |   Y    |   Y   |
+| Windows Server 2012        |   N    |   Y   |
+| Windows Server 2012 R2     |   N    |   Y   |
+| Windows Server 2016        |   N    |   Y   |
+
+
+
+| AIX          |  ppc32   |  ppc64   |
+|--------------|----------|----------|
+| AIX 7.1 TL4  |    Y     |    Y     |
+| AIX 7.2      |    Y     |    Y     |
+
+When public support for an operating system version ends, OpenJ9 can no longer be supported on that level.
+
+#### OpenJDK 8 build environment
+
+The following table shows the operating system and compilers that the project use to build and test OpenJ9 for different platforms:
 
 | Platform                                    | Operating system          |  Compiler                       |  Comments      |
 |---------------------------------------------|---------------------------|---------------------------------|----------------|
-| Linux x86 64-bit                            | Ubuntu 14.04              | gcc 4.4.7                       |                |
-| Linux on POWER<sup>&reg;</sup> LE 64-bit    | Ubuntu 14.04              | gcc 4.4.7                       |                |
-| Linux on IBM Z<sup>&reg;</sup> 64-bit       | Ubuntu 14.04              | gcc 4.4.7                       |                |
+| Linux x86 64-bit                            | Ubuntu 16.04              | gcc 4.4.7                       |                |
+| Linux on POWER<sup>&reg;</sup> LE 64-bit    | Ubuntu 16.04              | gcc 4.4.7                       |                |
+| Linux on IBM Z<sup>&reg;</sup> 64-bit       | Ubuntu 16.04              | gcc 4.4.7                       |                |
 | Windows x86 32-bit                          | Windows Server 2012       | Microsoft Visual Studio 2010 SP1|                |
```

Comment:
Either this or the 9/10 table is incorrect for the OS. We use the same machines to build 8/9/10.

Reply:
According to the machine labels https://ci.eclipse.org/openj9/, they are all Server 2012 R2 machine so the tables for 9/10 should be updated to reflect this.

Reply:
Changed all the Windows machines to be Windows Server 2012 R2. Hope that was what you intended.